css/ChannelInfo.less

```diff
@@ -33,7 +33,7 @@
 	border-radius: 6px;
 	text-align: center;
 	white-space: nowrap;
-	padding: 3px 9px;
+	padding: 3px 12px;
```

Comment:
Can't convert this one to 12px. it's too wide to allow the first two badges to fit on the same line, like in 1080.